fv/health_test.go

```diff
@@ -438,6 +439,51 @@ var _ = Describe("_HEALTH_ _BPF-SAFE_ health tests", func() {
 	})
 })
 
+var _ = infrastructure.DatastoreDescribe("Felix startup speed", []apiconfig.DatastoreType{apiconfig.EtcdV3 /*, apiconfig.Kubernetes*/}, func(getInfra infrastructure.InfraFactory) {
```

Comment:
Please ignore this test. It is just for the purpose of restarting felix and any my own testing. I have not figured out yet how to test the PR in fv.